memory-viz/src/automate.ts

```diff
@@ -73,7 +73,7 @@ function drawAutomatedStackFrames(stack_frames, configuration) {
         let width;
         let height;
 
-        if (stack_frame.name !== "BLANK") {
+        if (stack_frame.type !== "BLANK") {
```

Comment:
This change is out of scope of this PR (and in fact I suspect it's related to what @sarahsonder is working on currently).